### DISCUSSION TITLE
Last letter cut off when using keys set in context since v2.0.3?

### DISCUSSION THREAD

**Discussion by cletom** [Handlebars-Net/Handlebars.Net#499] [Q&A] [answered]
Hi,

I always used to get general info via Handlebars from context.

Therefore, I defined the values as follows:

```
var context = ImmutableDictionary<string, object>.Empty
                .Add("Name", "abcd");
```

and I was able to receive the value I set using

`{{.Name}}`

which worked fine until v 2.0.2.

When doing the same thing with v2.0.3 or later I always get an Error telling me `Nam is undefined` so it seems like somehow the last letter gets cut off since v2.0.3. If I add any sign, e.g. if I use `{{.Name1}}` i receive the value I set for "Name". I already debuged into my code and the context contains the correct key/value pairs.

Does anyone know what I have to do to make this work again or is there a bug in the library since v2.0.3?

Thanks

**Comment by oformaniuk** [comment, marked as answer]
Hello @cletom.
Yes, there's might be a bug. Try `this.Name`

**Reply by cletom** [reply]
Hi @zjklee,
thank you for your answer, `this.Name` works.
For first I'll rather stick with v2.0.2 and hope for a fix in one of the next releases, as I don't want to adapt my templates.
